onnxsim/onnx_simplifier.py

```diff
@@ -153,11 +155,16 @@ def is_dynamic(node):
     return copy.deepcopy(const_nodes)
 
 
-def forward(model, input_data: Dict[str, np.ndarray] = None,
-            input_shapes: Optional[TensorShapes] = None) -> Dict[str, np.ndarray]:
+def forward(model,
+            input_data: Optional[Tensors] = None,
+            input_shapes: Optional[TensorShapes] = None,
+            custom_lib: Optional[str] = None) -> Tensors:
     if input_shapes is None:
         input_shapes = {}
     sess_options = rt.SessionOptions()
+    if custom_lib is not None:
+        if os.path.exists(custom_lib):
```

Comment:
raise an exception if file named `custom_lib` does not exist

Reply:
that's great, i will add it